go.mod

```diff
@@ -5,7 +5,7 @@ go 1.18
 require (
 	github.com/CosmWasm/wasmd v0.24.0
 	github.com/cosmos/cosmos-proto v1.0.0-alpha7
-	github.com/cosmos/cosmos-sdk v0.45.6
+	github.com/cosmos/cosmos-sdk v0.46.0
```

Comment:
I think dev had some issues upgrading to v.46.0 

cc @ValarDragon

Reply:
Hmm, interesting. If this wasn't intentional then I'll revert on this - I just merged from main is why.

Reply:
I think it's fine leaving this as is (`v0.46.0`). Otherwise, it would get auto-updated every time someone runs `go mod tidy`. We use a replace directive to point to our sdk fork:
https://github.com/osmosis-labs/osmosis/blob/cf39ea388b71eeb5a8bf6de5befdd39ed66c2ea7/go.mod#L289

So this change has no effect. The canonical version being used is in the replace directive

Reply:
Aah i see we use a specific commit version